Sources/KlaviyoSwift/Klaviyo.swift

```diff
@@ -82,6 +82,11 @@ public struct KlaviyoSDK {
     public func resetProfile() {
         dispatchOnMainThread(action: .resetProfile)
     }
+    
+    public func resetBadgeCount(defaults: String) {
```

Comment:
```suggestion
    public func resetBadgeCount(defaults: UserDefaults) {
```
it would be safer to set the parameter type as `UserDefaults` so that the user can pass in an actual `UserDefaults` object rather than a typo-prone `String`. Then in your `KlaviyoReducer.reduce` method, your `resetBadgeCount` case would look like this:
```swift
case let .resetBadgeCount(defaults):
    defaults.set(0, forKey: "badgeCount")
    return .none
```

Reply:
yeah decided this will actually be moved into a plist value (that will be documented in the README, separate ticket) so this parameter will no longer be necessary